components/Icon.js

```diff
@@ -1,10 +1,11 @@
-const Icon = ({ name, tooltip, extraClasses = '' }) => (
+const Icon = ({ name, tooltip, extraClasses = '', onClick}) => (
```

Comment:
This component should not be modified - to add a click handler you should create a button that contains the icon. The icon itself should not be clickable

Reply:
Thanks! Made use of `IconButton` with adjusted padding instead. Slightly changes from the previous UI (with borders and colors) but I think it fits the theme more closely.